ReleaseNotes.md

```diff
@@ -1,5 +1,38 @@
 # Release Notes
 
+## New in Release 2.13.0
+
+Post-training Quantization:
+
+- Features:
+  - (OpenVINO) Implemented the use of the Scale Estimation algorithm to calculate quantization parameters within GPTQ for more accurate weight compression. The `gptq` and `scale_estimation` options can now be used together to calculate quantization parameters using the Scale Estimation algorithm within GPTQ in `nncf.compress_weights()`.
```

Comment:
Please combine items 1 and 3 to the following:
(OpenVINO) Added support for combining GPTQ with AWQ and Scale Estimation (SE) algorithms in `nncf.compress_weights()` for more accurate weight compression of LLMs. Thus, the following combinations with GPTQ are now supported: AWQ+GPTQ+SE, AWQ+GPTQ, GPTQ+SE, GPTQ.

Reply:
Updated